layouts/shortcodes/highlight-editable.html

```diff
@@ -0,0 +1,4 @@
+{{ $language := .Get 0 }}
+{{ $styling := .Get 1 }}
+
+{{- ((highlight .Inner $language $styling) | replaceRE `\$\$\$(.*)\$\$\$(.*)\$\$\$` `<span editable-id="${1}" class="editable-field" data-js-editable-field>${2}</span>` | safeHTML ) -}}
```

Comment:
can we apply it for regular `highlight`? in this case no need to rename and think about `highlight` vs `highlight-editable`

Reply:
Unfortunately we can't do that. The highlight in this case is built-in.

Reply:
ah ok